kuma/wiki/tests/test_models.py

```diff
@@ -601,19 +601,24 @@ def test_correct_based_on_to_current_revision(self):
     def test_previous(self):
         """Revision.previous should return this revision's document's
         most recent approved revision."""
-        rev = revision(is_approved=True, save=True)
-        eq_(None, rev.previous)
-        # wait a second so next revision is a different datetime
-        time.sleep(1)
+        rev = revision(is_approved=True, created=datetime(2017, 4, 15, 9, 23),
+                       save=True)
         next_rev = revision(document=rev.document, content="Updated",
```

Comment:
Can you create more than one ``revision`` so there are another revision exist. So if our code break it can be cought?